<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectionDialog</class>
 <widget class="QDialog" name="DirectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direction</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QDial" name="dial">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchTarget">
      <double>2.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>z-rotation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="z_value">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>179</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="x_value">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="y_value">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>y</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DirectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DirectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>z_value</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>z_value</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
